llm/backfill_summaries.py

```diff
@@ -0,0 +1,82 @@
+# This script fills any missing 'summary' or 'topics' fields on the data model.
+# The document must have a 'Title' and 'DocumentText' field to generate them.
+#
+# Developer notes:
+# - you'll need to set the 'OPENAI_API_KEY' environment variable
+import firebase_admin
+from llm_functions import get_summary_api_function, get_tags_api_function_v2
+from firebase_admin import firestore
+from bill_on_document_created import get_categories_from_topics, CATEGORY_BY_TOPIC
+import csv
+from normalize_summaries import normalize_summary
+
+# Application Default credentials are automatically created.
```

Comment:
Do we have docs on how to connect to the MAPLE prod firebase, assuming that's what you are doing? If so, can we link that here?

Reply:
Great! Can we link here?